test/us12_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import datetime
+import sys
+import os
+sys.path.append(os.path.abspath('../src'))
+import us12
+
+class us12_test(unittest.TestCase):
+    def test1(self):
+        filename = "seeds/test121.ged" 
+        ret = ['I2']
+        self.assertEqual(us12.us12(filename),ret)
```

Comment:
if you use the import in the previous comment ya could just do `us12()` instead of `us12.us12()` just a neat trick